goalgorilla/drupal_social
# DS 356 Updated docker-compose syntax to version 2.0

Support for the new docker-compose syntax.

How to test:

1. Remove all social docker containers ``docker-compose down``

2. Checkout this branch

3. Run ``docker-compose up -d`` and check that all containers are started

4. Install the site and check that everything still works without errors

5. Potentially check versions of Docker and Docker compose on travis and on the Hernzer CI server

6. After the merge notify developers in slack to recreate the containers and do a fresh install (needed because of the new volume management)
